README.md

```diff
@@ -71,195 +70,190 @@ are available. Please click on the links below for more informations and picture
+- (Optional) Install [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+  as plugin (to have advanced Git features in VSCode).
+- Now, that the Git repository is saved locally, import this folder into Platformio as a project.
 - Select the [desired
   environment](https://forum.espuino.de/t/projekt-und-profilwechsel-in-visual-studio-code/768) (e.g.
   lolin_d32_pro_sdmmc_pe).
 - Edit `src/settings.h` according your needs.
 - Edit board-specific (`HAL`) config-file (e.g. `settings-lolin_d32_pro_sdmmc_pe.h` for Lolin
   D32/D32 pro). If you're running a board that is not listed there: start with `settings-custom.h`
   and change it according your needs.
-- Connect your develboard via USB, click the alien-head to the left, choose the project-task that
-  matches your desired HAL and run `Upload and Monitor`. All libraries necessary should be fetched
-  in background now followed by code-compilation. After that, your ESP32 is flashed with the
-  firmware. Depending on your develboard it might me necessary to push a button in order to allow
-  ESP32 to enter flashmode (not necessary für Lolin32, D32 und D32 pro).
-- Now have a look at the serial-output at the bottom of Visual Studio Code's window. At the first
-  run there might appear a few error-messages (related to missing entries in NVS). Don't worry, this
-  is just normal. However, make sure SD is running as this is mandatory!
+- Connect your develboard via USB, click the alien-head icon in the left sidebar, choose the
+  project-task that matches your desired HAL and run `Upload and Monitor`. All libraries necessary
+  should be fetched in background now followed by code-compilation. After that, your ESP32 is
+  flashed with the firmware. Depending on your develboard it might be necessary to push a button in
+  order to allow ESP32 to enter flash mode (not necessary für Lolin32, D32 und D32 pro).
+- Now have a look at the serial output at the bottom of Visual Studio Code's window. At the first
+  run there might appear a few error messages (related to missing entries in NVS). Don't worry, this
+  is just normal. However, make sure the SD card is detected as this is mandatory!
 - If everything ran fine, at the first run, ESPuino should open an access-point with the name
   "ESPuino". Join this WiFi with your computer (or mobile) and enter `http://192.168.4.1` to your
-  webbrowser. Enter WiFi-credentials and the hostname. After saving the configuraton, restart
+  webbrowser. Enter WiFi credentials and the hostname. After saving the configuration, restart
   ESPuino.
 - After reboot ESPuino tries to join your WiFi (with the credentials previously entered). If that
-  was successful, an IP is shown in the serial-console. You can call ESPuino's GUI using a
-  webbrowser via this IP; make sure to allow Javascript. If mDNS-feature is active in
+  was successful, an IP is shown in the serial console. You can call ESPuino's GUI using a
```

Comment:
While we're at it, I'd say change 'call' to 'reach' or 'access'. You don't call a GUI.